packages/lesswrong/components/posts/PostsPage/PostsPageTitle.tsx

```diff
@@ -46,7 +46,9 @@ const styles = defineStyles("PostsPageTitle", (theme: ThemeType) => ({
     ...postPageTitleStyles(theme)
   },
   draft: {
-    color: theme.palette.text.dim4
+    color: theme.palette.text.dim4,
+    fontSize: isFriendlyUI ? "1em" : "0.5em",
+    display: isFriendlyUI ? "inline" : "block",
```

Comment:
These can just be `undefined` for the `isFriendlyUI` case if the intent is to avoid changing what it looks like in that case